modules/backup/variables.tf

```
@@ -93,3 +93,9 @@ variable "compress_export" {
   type        = bool
   default     = true
 }
+
+variable "unique_suffix" {
+  description = "Unique suffix to add to scheduler jobs and workflows names."
+  type        = string
+  default     = "0"
```

Any reason we need to default to 0? Usually empty is a good default for  optional suffix.
```suggestion
  default     = ""
```

---

If empty is okay, we should construct the name in main.tf also in such a way that the trailing `-` does not occur - `var.default != "" ? "sql-export-${var.sql_instance}-${var.unique_suffix}" : "sql-export-${var.sql_instance}"`

---

No particular reason - I just thought it looked nicer by not crunching the suffix against the instance name. I'm happy either way though - changed it to an empty default :)